moz_minions/minions.py

```diff
@@ -51,11 +54,16 @@ def __init__(self, name, **kwargs):
         if 'command' in kwargs:
```

Comment:
I'll leave the validator here and head to write tests in another pr.  Supposedly they will be all converged.